### ISSUE TITLE
Events are broken since choo implementation

### ISSUE THREAD

**Issue by ritchse** [hydra-synth/hydra#185]
Example from pressing Ctrl+Shift+S

```
17:24:17 🚨 choo No listeners for screencap
bundle.min.js?1.3.0:161 17:24:17 ✨ choo screencap 0ms
```

Same happens with Ctrl+Shift+L, for example

Issue also reported by obakegaku on the discord

**Comment by ojack** [comment]
Are these the only two commands that are not working?

**Comment by ritchse** [comment]
Just checked all of them, that seems to be the case. btw while you're at it we could differentiate save screenshot and save .js file as we'd talked ? iirc we talked about keeping Ctrl+Shift+S as save screenshot and doing Ctrl+Alt+Shift+S for the .js file.

**Comment by ojack** [comment]
fixed in v1.3.3
